src/RequestHandler/outofprocess/forwardinghandler.h

```diff
@@ -174,11 +174,33 @@ class FORWARDING_HANDLER : public REQUEST_HANDLER
     BOOL                                m_fResponseHeadersReceivedAndSet;
     BOOL                                m_fResetConnection;
     BOOL                                m_fHandleClosedDueToClient;
-    BOOL                                m_fFinishRequest;
-    BOOL                                m_fHasError;
     BOOL                                m_fDoReverseRewriteHeaders;
+    BOOL                                m_fServerResetConn;
```

Comment:
volatile for m_fServerResetConn as well?

Reply:
no need. it's inside a lock